motor_control/throttle/README.md

```diff
@@ -0,0 +1,23 @@
+# Welcome to rEV!
```

Comment:
Hm is there a reason the root README is copied into here? Might be more useful to have more specifics about the throttle API here.